src/libs/Middleware/Reauthentication.js

```diff
@@ -113,6 +111,15 @@ function Reauthentication(response, request, isFromSequentialQueue) {
 
             // Return response data so we can chain the response with the following middlewares.
             return data;
+        })
+        .catch((error) => {
+            // If the request is on the sequential queue, re-throw the error so we can decide to retry or not
+            if (isFromSequentialQueue) {
+                throw error;
+            }
+
```

Comment:
I don't think we need to log here because we should have already logged in the logging middleware. I know that we had this logging block in the Retry middleware, but I don't feel like it provides a ton of value. It will only log for requests that are not on the SequentialQueue, which is kind of odd. https://github.com/Expensify/App/blob/d238b2368357f8954f1fb290966975a9c3792572/src/libs/Middleware/Retry.js#L29-L33

If we want to keep these generic error log lines I think we should move them into the logging middleware. Logging when the Log api command fails does seem helpful but again I don't know if it's necessary. Thoughts?